orca-front50/src/main/java/com/netflix/spinnaker/orca/front50/pipeline/PipelineStage.java

```diff
@@ -34,13 +34,17 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 @Component
 public class PipelineStage implements StageDefinitionBuilder, CancellableStage {
 
   private final Logger log = LoggerFactory.getLogger(getClass());
 
+  @Value("${defaultSkipDownstreamOutput:false}")
```

Comment:
makes sense in principle, but if possible I think it would be better to avoid introducing a new top-level config. Maybe it can be namespaced, e.g.:

```suggestion
  @Value("${stages.pipeline.defaultSkipDownstreamOutput:false}")
```

Reply:
done